src/stories/assets/question/simple-single-tree-select.json

```diff
@@ -0,0 +1,85 @@
+{
+  "@id": "ma-vlastnika-section-777",
+  "@type": "http://onto.fel.cvut.cz/ontologies/documentation/question",
+  "http://onto.fel.cvut.cz/ontologies/form-layout/has-layout-class": "type-ahead",
+  "http://onto.fel.cvut.cz/ontologies/form/has-non-selectable-value": "fyzicka-osoba",
+  "http://onto.fel.cvut.cz/ontologies/form/has-possible-value": [
+    {
+      "@id": "fyzicka-osoba",
+      "http://www.w3.org/2000/01/rdf-schema#label": "Fyzická osoba",
+      "http://www.w3.org/2002/07/owl#disjointWith": {
```

Comment:
It does not make sense to have disjoint values

Reply:
remove all of those references in this file

Reply:
Done